### ISSUE TITLE
В авансовый отчет добавить режим редактирования проведенного документа.

### ISSUE THREAD

**Issue by Anestezia** [a2develop/bugTracker#1359]
Там все равно нет номенклатуры. Единственное, что может упереться -- это дата позже накладной (акта), и то, смотря как посмотреть. ведь, чтоб отчитаться, дается несколько дней (в разных случаях по-разному), а товар может быть уже куплен.

Login: buhgalter.ogp@gmail.com
Добрий день! Було б добре, щоб можна було коригувати документи без розпроведення інших. Наприклад, мені потрібно виправити помилку в авансовому звіті № 9. Для цього потрібно його розпровести. А я не можу цього зробити, бо за ним вже проведені 22 авансових звіти. То мені потрібно розпровести їх усі, потім виправити помилку, потім провести всі. Це ж мій час. Зробіть, будь-ласка, щоб виправляти помилки було простіше. Дякую

SE-2229 Додаю до попереднього звернення: А для того, щоб мені розпровести 22 авансових звіти, потрібно розпровести і ВИДАЛИТИ !!! ВСІ !!! ПРИБУТКОВІ НАКЛАДНІ, які прив'язані до цих звітів!!! Зробіть, будь ласка, щоб можна було розпроводити і виправляти помилки БЕЗ ВИДАЛЕННЯ зв'язаних документів. А помилки в авансових звітах, щоб можна було виправляти, без розпроведення інших. Тому що це дуже не зручно і забирає багато часу.

**Comment by htrofymova** [comment]
Все-таки хотелось бы уточнить, какие именно ошибки хотят исправлять.

**Comment by Anestezia** [comment]
Так, в авансовому звіті помилка. І я хочу її виправити. Але для того, щоб розпровести цей авансовий звіт, мені потрібно розпровести і всі наступні таким же чином - розпроводячі і видаляючі всі привязані до них документи!!! А їх 69. Цей звіт - 31-й!
Про це я і кажу. Я уже зверталась до вас із такою проблемою. Чому вона не вирішується? Навіщо розпроводити і видаляти наступні документи?
Чи є простіший спосіб виправити свою помилку? Він має бути. (Это по поводу того авансового, по которому не было проводки по 631 счету )
